versioned_hdf5/wrappers.py

```diff
@@ -221,21 +232,50 @@ def create_dataset(
         data: ArrayLike | None = None,
         fillvalue: Any | None = None,
         chunks: tuple[int, ...] | int | bool | None = None,
-        compression=None,
-        compression_opts=None,
+        # Filters
+        compression: Any | None | Default = DEFAULT,
+        compression_opts: Any | None | Default = DEFAULT,
+        scaleoffset: int | None | Default = DEFAULT,
+        shuffle: bool | Default = DEFAULT,
+        fletcher32: bool | Default = DEFAULT,
+        # Ignored (with a warning)
         **kwds: Any,
     ):
         self._check_committed()
-        dirname = posixpath.dirname(name)
-        if dirname and dirname not in self:
-            self.create_group(dirname)
 
+        # Disregard ignored parameters with a warning
         for k, v in kwds.items():
-            if v is not None and not (k == "maxshape" and all(i is None for i in v)):
+            if v is not None:
+                if k == "maxshape" and isinstance(v, tuple) and set(v) == {None}:
+                    continue
                 warnings.warn(
                     f"The {k} parameter is currently ignored for versioned datasets.",
                     stacklevel=2,
                 )
+        del kwds
```

Comment:
Why do we do this here? `kwds` is not used further down, right? I guess this just removes the temptation to use this later on?

Reply:
Correct, it's just for code sanity.